src/esm_calendar/esm_calendar.py

```diff
@@ -895,3 +895,38 @@ def sub_tuple(self, to_sub):
 
         new_date = self.from_list(result)
         return new_date
+
+    def is_in_interval(self, interval, sep="->") -> bool:
```

Comment:
```suggestion
    def is_in_interval(self, interval) -> bool:
```

You probably can make a type annotation of `interval: Tuple[Date, Date]`, but I can't remember the exact syntax